src/www/ui/ui-report-conf.php

```diff
@@ -321,7 +321,8 @@ function (&$licArray, $obTopic) use (&$obLicenses) {
       $columns = "";
       foreach ($this->mapDBColumns as $key => $value) {
         $columns .= $value." = $".$i.", ";
-        $parms[] = pg_escape_string(GetParm($key, PARM_RAW));
+        $escaped = pg_escape_string(GetParm($key, PARM_RAW));
+        $parms[] = str_replace("''", "'", $escaped);
```

Comment:
Do you think `pg_escape_string()` is required here? As the parameters are passed in the `column = $1` format and will be escaped in the pg library.

Because performing the `str_replace()` will defeat the purpose of calling `pg_escape_string()`.